### PR TITLE
Fix show error for checkbox, radio, checkboxList, radioList

### PR DESCRIPTION
**Pull request by egrekov**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #173

### CODE REVIEW

**Comment by bizley** [comment]
Can we get some unit tests for this?

**Comment by samdark** [comment]
Looks like tests are failing...

**Comment by egrekov** [comment]
> Looks like tests are failing...

Yes, I'll deal with the tests a little later, it's bad that they don't run locally, it would be easier.

**Comment by samdark** [comment]
Don't run locally? Why?

**Comment by samdark** [comment]
Update from master. It may help (in case you're under Windows).

**Comment by egrekov** [comment]
root@516debc04af8:/opt/app# vendor/phpunit/phpunit/phpunit --filter ActiveFieldTest
PHPUnit 4.8.34 by Sebastian Bergmann and contributors.

Runtime:        PHP 7.3.27
Configuration:  /opt/app/phpunit.xml.dist

.............

Time: 2.62 seconds, Memory: 8.00MB

OK (13 tests, 13 assertions)

File: src/ActiveField.php [hunk 1]
```diff
@@ -342,6 +344,7 @@ public function checkboxList($items, $options = [])
                     Html::addCssClass($wrapperOptions, 'custom-control-inline');
                 }
 
+                $this->addErrorClassIfNeeded($options);
```
Comment (bizley) [inline src/ActiveField.php:347]: Should it be here or later on (before line 359)?

File: src/ActiveField.php [hunk 2]
```diff
@@ -383,6 +386,7 @@ public function radioList($items, $options = [])
...
```
Comment (bizley) [inline src/ActiveField.php:389]: Should it be here or later on (before line 401)?

File: src/ActiveField.php (hunk 1 above)
Comment (egrekov) [reply src/ActiveField.php:347]: It should be here, checkboxList contains several values and should be added here for correct display.

File: src/ActiveField.php (hunk 2 above)
Comment (egrekov) [reply src/ActiveField.php:389]: It should be here, radioList contains several values and should be added here for correct display.

File: src/ActiveField.php (hunk 1 above)
Comment (egrekov) [reply src/ActiveField.php:347]: 347 line

File: src/ActiveField.php (hunk 2 above)
Comment (egrekov) [reply src/ActiveField.php:389]: 389 line

**Review by bizley — APPROVED** [review]

**Comment by bizley** [comment]
Please add CHANGELOG entry and it should be good to go.

**Comment by egrekov** [comment]
> Please add CHANGELOG entry and it should be good to go.
And how to write correctly and in which block?
```
-2.0.10 under development
+2.0.10 March 04, 2021
 ------------------------
 
 - Enh #205: Added `Toast` widget (simialbi)
-
+- Bug #173: Fix show error for checkbox, radio, checkboxList, radioList
 
 2.0.9 November 10, 2020
 -----------------------
```

**Comment by bizley** [comment]
Don't change the date, just add new line under `Enh #205: Added Toast widget (simialbi)` with something like:
`- Bug #173: Fix bug with error messages not being displayed for checkbox, radio, checkboxList, and radioList (egrekov)`

**Comment by egrekov** [comment]
Ok, CHANGELOG updated

**Comment by samdark** [comment]
Thanks!

**Pull request merged**